kubetest2-tf/deployer/deployer.go

```diff
@@ -76,10 +78,10 @@ type deployer struct {
 
 	RepoRoot              string            `desc:"The path to the root of the local kubernetes repo. Necessary to call certain scripts. Defaults to the current directory. If operating in legacy mode, this should be set to the local kubernetes/kubernetes repo."`
 	IgnoreClusterDir      bool              `desc:"Ignore the cluster folder if exists"`
-	AutoApprove           bool              `desc:"Terraform Auto Approve"`
+	AutoApprove           bool              `desc:"Auto-approve the deployment of infrastructure through terraform" flag:",deprecated"`
```

Comment:
Since `auto-approved` is no longer supported, setting/unsetting will not have any effect on the execution.

Reply:
Created an issue on the terraform exec repo just to get an opinion!